File: LibraryAPI/src/schemas/TakingBook/taking_book_update.py
from pydantic import BaseModel
from datetime import date


class TakingBookUpdate(BaseModel):
    book_id: int | None = None
    user_id: int | None = None
    employee_id: int | None = None
    taking_date: date | None = None
    is_returned: bool | None = None
    return_date: date | None = None

File: LibrarryApp/LibrarryDesktop/Models/ApiRequestModels/TakingBookUpdate.cs
using Newtonsoft.Json;

namespace LibrarryDesktop.Models.ApiRequestModels
{
    public class TakingBookUpdate
    {
        [JsonProperty("book_id")]
        public int? BookId { get; set; }
        [JsonProperty("user_id")]
        public int? UserId { get; set; }
        [JsonProperty("employee_id")]
        public int? EmployeeId { get; set; }
        [JsonProperty("taking_date")]
        public DateTime? TakingDate { get; set; }
        [JsonProperty("is_returned")]
        public bool? IsReturned { get; set; }
        [JsonProperty("return_date")]
        public DateTime? ReturnDate { get; set; }
    }
}